src/server/server_extensions/__tests__/add_memoized_factory_to_request.js

```diff
@@ -27,47 +26,46 @@ describe('server.addMemoizedFactoryToRequest()', () => {
 
   it('throws when propertyName is not a string', () => {
     const { server } = setup();
-    expect(() => server.addMemoizedFactoryToRequest()).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest(null)).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest(1)).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest(true)).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest(/abc/)).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest(['foo'])).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest([1])).to.throwError('propertyName must be a string');
-    expect(() => server.addMemoizedFactoryToRequest({})).to.throwError('propertyName must be a string');
+    expect(() => server.addMemoizedFactoryToRequest()).toThrowError('methodName must be a string');
```

Comment:
This was also a buggy assertion before. No `propertyName must be` exists in Kibana, only `methodName must be`